fluentd/run.sh

```diff
@@ -286,7 +286,8 @@ fi
 # pods unable to be terminated because fluentd has them busy
 if [ "${USE_MUX:-}" = "true" ] ; then
     : # skip umount
-else
+elif [ "${USE_CRIO:false}" = false ] ; then
```

Comment:
If you put a `mount` command here, before the umount, what does it show is mounted?

Reply:
@richm, this is the output from mount...
```
overlay on / type overlay (rw,relatime,seclabel,lowerdir=/var/lib/containers/storage/overlay/l/BUCTIFKU7M76YGJS5HBB4LSYKB:/var/lib/containers/storage/overlay/l/7GJOSIMXZU7MRJ7WLMVCBSB7G2,upperdir=/var/lib/containers/storage/overlay/b3860d61851018e2b701c6d3bfd0e536270d2e16d4175ee9574eebe7a6bbb51e/diff,workdir=/var/lib/containers/storage/overlay/b3860d61851018e2b701c6d3bfd0e536270d2e16d4175ee9574eebe7a6bbb51e/work)
/run/containers/storage/overlay-containers/e4bab807fe9084e54049a29a8886dff70d4961aaf1c7ff79327f8351359e67ce/userdata/proc on /proc type proc (rw,relatime)
/run/containers/storage/overlay-containers/e4bab807fe9084e54049a29a8886dff70d4961aaf1c7ff79327f8351359e67ce/userdata/tmpfs on /dev type tmpfs (rw,nosuid,seclabel,size=65536k,mode=755)
/run/containers/storage/overlay-containers/e4bab807fe9084e54049a29a8886dff70d4961aaf1c7ff79327f8351359e67ce/userdata/devpts on /dev/pts type devpts (rw,nosuid,noexec,relatime,seclabel,gid=5,mode=620,ptmxmode=666)
/run/containers/storage/overlay-containers/e4bab807fe9084e54049a29a8886dff70d4961aaf1c7ff79327f8351359e67ce/userdata/mqueue on /dev/mqueue type mqueue (rw,nosuid,nodev,noexec,relatime,seclabel)
/run/containers/storage/overlay-containers/e4bab807fe9084e54049a29a8886dff70d4961aaf1c7ff79327f8351359e67ce/userdata/sysfs on /sys type sysfs (ro,nosuid,nodev,noexec,relatime,seclabel)
tmpfs on /sys/fs/cgroup type tmpfs (rw,nosuid,nodev,noexec,relatime,seclabel,mode=755)
cgroup on /sys/fs/cgroup/systemd type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,xattr,release_agent=/usr/lib/systemd/systemd-cgroups-agent,name=systemd)
cgroup on /sys/fs/cgroup/pids type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,pids)
cgroup on /sys/fs/cgroup/net_cls,net_prio type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,net_prio,net_cls)
cgroup on /sys/fs/cgroup/blkio type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,blkio)
cgroup on /sys/fs/cgroup/cpu,cpuacct type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,cpuacct,cpu)
cgroup on /sys/fs/cgroup/freezer type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,freezer)
cgroup on /sys/fs/cgroup/memory type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,memory)
cgroup on /sys/fs/cgroup/devices type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,devices)
cgroup on /sys/fs/cgroup/cpuset type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,cpuset)
cgroup on /sys/fs/cgroup/perf_event type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,perf_event)
cgroup on /sys/fs/cgroup/hugetlb type cgroup (rw,nosuid,nodev,noexec,relatime,seclabel,hugetlb)
shm on /dev/shm type tmpfs (rw,nosuid,nodev,noexec,relatime,seclabel,size=65536k)
tmpfs on /etc/resolv.conf type tmpfs (rw,nosuid,nodev,noexec,seclabel,mode=755)
tmpfs on /etc/hostname type tmpfs (rw,nosuid,nodev,seclabel,mode=755)
tmpfs on /run/secrets type tmpfs (rw,nosuid,nodev,seclabel,mode=755)
/dev/xvda2 on /var/log type xfs (rw,relatime,seclabel,attr2,inode64,noquota)
/dev/xvda2 on /etc/hosts type xfs (rw,relatime,seclabel,attr2,inode64,noquota)
tmpfs on /run/secrets type tmpfs (rw,nosuid,nodev,seclabel,mode=755)
/dev/xvda2 on /dev/termination-log type xfs (rw,relatime,seclabel,attr2,inode64,noquota)
/dev/xvda2 on /etc/docker-hostname type xfs (ro,relatime,seclabel,attr2,inode64,noquota)
/dev/xvda2 on /usr/share/zoneinfo/UTC type xfs (ro,relatime,seclabel,attr2,inode64,noquota)
/dev/xvda2 on /etc/docker type xfs (ro,relatime,seclabel,attr2,inode64,noquota)
/dev/xvda2 on /etc/origin/node type xfs (ro,relatime,seclabel,attr2,inode64,noquota)
/dev/xvda2 on /etc/sysconfig/docker type xfs (ro,relatime,seclabel,attr2,inode64,noquota)
/dev/xvda2 on /var/lib/fluentd type xfs (rw,relatime,seclabel,attr2,inode64,noquota)
tmpfs on /etc/fluent/muxkeys type tmpfs (ro,relatime,seclabel)
tmpfs on /etc/fluent/keys type tmpfs (ro,relatime,seclabel)
/dev/xvda2 on /var/lib/docker type xfs (ro,relatime,seclabel,attr2,inode64,noquota)
/dev/xvda2 on /var/lib/docker/containers type xfs (rw,relatime,seclabel,attr2,inode64,noquota)
tmpfs on /run/log/journal type tmpfs (rw,nosuid,nodev,seclabel,mode=755)
/dev/xvda2 on /etc/fluent/configs.d/user type xfs (ro,relatime,seclabel,attr2,inode64,noquota)
tmpfs on /run/secrets/kubernetes.io/serviceaccount type tmpfs (ro,relatime,seclabel)
```
And the directory is empty contrary to the docker case.
```
# ls -l /var/lib/docker/containers
total 0
```